lighthouse-core/audits/bootup-time.js

```diff
@@ -73,22 +74,25 @@ class BootupTime extends Audit {
       ];
 
       // map data in correct format to create a table
-      const results = Array.from(executionTimings).map(([url, groups]) => {
-        // Add up the totalBootupTime for all the taskGroups
-        totalBootupTime += Object.keys(groups).reduce((sum, name) => sum += groups[name], 0);
-        extendedInfo[url] = groups;
+      const results = Array.from(executionTimings)
+        .map(([url, groups]) => {
+          // Add up the totalBootupTime for all the taskGroups
+          totalBootupTime += Object.keys(groups).reduce((sum, name) => sum += groups[name], 0);
```

Comment:
@paulirish should we create an "Other" entry in the table so the total still adds up?

Reply:
ehhhh we _could_. wdyt?

Reply:
I'm in favor of "Other" unless there are better ideas

Reply:
I looked into this and the table already doesn't sum to the total time displayed. the total time shown at the top includes all non-script time (layouts/paints/etc)

so if we really wanted to also capture those, we might do something like 

![image](https://user-images.githubusercontent.com/39191/34752871-abf4178c-f569-11e7-97dc-21b87a14de1d.png)

but it's kinda wonky.  

I'm in favor of skipping the "other" row(s)

Reply:
Alright that's fine we can skip too

Reply:
So what's the advice here? 😋